proto/detectors.proto

```diff
@@ -977,6 +977,17 @@ enum DetectorType {
   Ngrok = 966;
   Replicate = 967;
   Postgres = 968;
+  AzureActiveDirectoryApplicationSecret = 969;
+  AzureCacheForRedisAccessKey = 970;
+  AzureCosmosdbKeyIdentifiable = 971;
+  AzureDevopsPersonalAccessToken = 972;
+  AzureFunctionKey = 973;
+  AzureMlWebServiceClassicIdentifiableKey = 974;
+  AzureSasToken = 975;
+  AzureSearchAdminKey = 976;
+  AzureSearchQueryKey = 977;
+  AzureManagementCertificate = 978;
+  AzureSql = 979;
```

Comment:
nit: Similarly here, lets make `Sql` -> `SQL` so it matches our other detector's casing. (ex: `SQLServer`)